src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -242,6 +243,15 @@ export class HuiCardPicker extends LitElement {
           align-items: center;
           justify-content: center;
         }
+
+        .overlay {
+          position: absolute;
+          top: 0;
+          left: 0;
+          width: 100%;
+          height: 100%;
+          z-index: 1;
```

Comment:
not needed I think?

Reply:
This is needed. Otherwise I can still toggle